File: exer9.pas
Program AreaTriangulo ;

Var
	lado1, lado2, lado3, semiper, area: real;

 Begin
      writeln('Dado os lados do triângulo, calcula a sua respectiva área');
      writeln('Digite o comprimento do primeiro lado do triângulo:');
      readln(lado1);
      
      writeln('Digite o comprimento do segundo lado do triângulo:');
      readln(lado2);
      
      writeln('Digite o comprimento do terceiro lado do triângulo:');
      readln(lado3);
      
      semiper := (lado1 + lado2 + lado3)/2;
      
      area := sqrt(semiper * (semiper - lado1) * (semiper - lado2) * (semiper - lado3));
      
      writeln('A área do triângulo de lado com as medidas fornecidas é de:', area);
 End.

File: exer9.c
/**
 * Exercício 9, Lista de Exercícios 06-03-2013
 * Programa que calcula a área de um triângulo
 */

#include <stdio.h>
#include <math.h>

int main()
{

	float lado1, lado2, lado3, semiper, area;

	printf("%s\n", "Dado os lados do triângulo, calcula a sua respectiva área");
	printf("%s\n", "Digite o comprimento do primeiro lado do triângulo:");
	scanf("%f", &lado1);

	printf("%s\n", "Digite o comprimento do segundo lado do triângulo:");
	scanf("%f", &lado2);

	printf("%s\n", "Digite o comprimento do terceiro lado do triângulo:");
	scanf("%f", &lado3);

	/**
	 * Cálculo do semiperimetro
	 */
	semiper = (lado1 + lado2 + lado3)/2;

	/**
	 * Cálculo da área por semiperímetro
	 */
	area = sqrt(semiper * (semiper - lado1) * (semiper - lado2) * (semiper - lado3));

	printf("%s: %f\n", "Área do triângulo dos lados fornecidos é de", area);

	return 0;
}